Formula/kubevious.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class Kubevious < Formula
+  desc "Detects and prevents Kubernetes misconfigurations and violations"
+  homepage "https://github.com/kubevious/kubevious"
+  url "https://registry.npmjs.org/kubevious/-/kubevious-1.0.20.tgz"
+  sha256 "04331ac58518ce646e039203ee3c825e5b1f9e326df0dee9c7a6df7335650304"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://registry.npmjs.org/kubevious/latest"
+    regex(/["']version["']:\s*?["']([^"']+)["']/i)
+  end
+
+  depends_on "node@14"
```

Comment:
Why not a modern node?

Reply:
There are a few dependent libraries used that didn't work well with the way Promises works in node v18 and above. Will have this resolved until the security support expires (April 30, 2023).

Reply:
Any upstream issue tracker for this? Also worth adding a code comment as well.

Reply:
No, just the way Promises are handled is more strict in later versions. Created an issue to track: https://github.com/kubevious/cli/issues/8